addons/binding/org.openhab.binding.ftpupload/ESH-INF/thing/thing-types.xml

```diff
@@ -28,24 +28,24 @@
 
     <channel-type id="image-channel">
         <item-type>Image</item-type>
-        <label>Camera image</label>
-        <description>Image from network camera</description>
+        <label>Image</label>
+        <description>Image received via FTP</description>
         <state readOnly="true"></state>
     </channel-type>
-    <channel-type id="motion-trigger">
+    <channel-type id="image-received">
```

Comment:
Does this channel really check if the received file is an image? I see in your later commit you renamed the `thing` to `imagereceiver`, so you probably want to check in your implementation that the received file is an image file.

Reply:
Yes, I renamed thing to imagereceiver as only supported channel is a image channel. I also thought trigger channel name and in the beginning it was "file-received", but then I ended up to image-received because of thing name.